Jenkinsfile

```diff
@@ -71,7 +71,7 @@ pipeline {
             }

```

Comment:
Can't comment on the line itself, but could you remove the guff from the branch name in the line above so that `uar-test` will get deployed to the environment when built?